dynawo/sources/Models/Modelica/Dynawo/Electrical/Controls/Machines/PowerSystemStabilizers/Standard/PssKundur.mo

```diff
@@ -0,0 +1,69 @@
+within Dynawo.Electrical.Controls.Machines.PowerSystemStabilizers.Standard;
+
+/*
+* Copyright (c) 2022, RTE (http://www.rte-france.com)
+* See AUTHORS.txt
+* All rights reserved.
+* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, you can obtain one at http://mozilla.org/MPL/2.0/.
+* SPDX-License-Identifier: MPL-2.0
+*
+* This file is part of Dynawo, an hybrid C++/Modelica open source suite
+* of simulation tools for power systems.
+*/
+
+model PssKundur "Power system stabilizer based on Kundur's book"
+  import Modelica;
+  import Dynawo.Electrical.SystemBase;
+
+  parameter Types.PerUnit KStab "PSS gain";
+  parameter Types.Time t1 "Phase compensation time constant in s";
+  parameter Types.Time t2 "Phase compensation time constant in s";
```

Comment:
Same name for t1 and t2 ?

Reply:
parameter Types.Time t1 "Phase compensation lead time constant in s";
  parameter Types.Time t2 "Phase compensation lag time constant in s";